src/org/daisy/dotify/formatter/impl/row/SegmentProcessor.java

```diff
@@ -1127,17 +1123,7 @@ private String getLeftIndentWithLabel(String listLabel, int indent) {
                     preTabPos > leaderPos ||
                     offset - align < 0
                 ) { // if tab position has been passed or if text does not fit within row, try on a new row
-                    MarginProperties leftMargin = currentRow.getLeftMargin();
-                    if (hasCurrentRow()) {
-                        ret = flushCurrentRow();
-                    }
-                    newCurrentRow(leftMargin, processorContext.getMargins().getRightMargin());
-                    row = new RowInfo(
-                        getLeftIndent(nextRowIndent),
-                        processorContext.getAvailable() - lineProps.getReservedWidth()
-                    );
-                    //update offset
-                    offset = leaderPos - row.getPreTabPosition(currentRow);
+                    return Optional.ofNullable(flushCurrentRow());
```

Comment:
Does this also work if the current row is the last row on a page? (and then there should be a flush on a new page)

Reply:
Not sure I understand.